src/domain/orders/new/components/items.tsx

```diff
@@ -32,19 +32,27 @@ const Items = () => {
     form: { control, register, setValue, getValues },
   } = useNewOrderForm()
 
-  const { fields, append, remove } = items
+  const { fields, append, remove, update } = items
 
   const [editQuantity, setEditQuantity] = useState(-1)
   const [editPrice, setEditPrice] = useState(-1)
 
   const layeredContext = useContext(LayeredModalContext)
 
-  const addItem = (variants: PricedVariant[]) => {
+  const addItem = async (variants: ProductVariant[]) => {
     const ids = fields.map((field) => field.variant_id)
+
     const itemsToAdd = variants.filter((v) => !ids.includes(v.id))
 
+    const variantIds = itemsToAdd.map((v) => v.id)
+
+    const { data } = (await Medusa.variants.list({
```

Comment:
nit (non-blocking): I think we should look to phase out the old API service, so I am not too keen on adding the use of it to a new file. Could we use medusajs instead?

Reply:
@pKorsholm Do you have any comments on this? :)

Reply:
changed it 😄